### PR TITLE
Roll to build tools 30.0.2

### PR DESCRIPTION
**Pull request by dnfield**
Allows relanding https://github.com/flutter/engine/pull/27607

### CODE REVIEW

**Comment by blasten** [comment]
@chinmaygarde do you know when we can expect merging buildroot into the engine? I also have some pending changes, but I was waiting for this merge to happen first.

**Comment by dnfield** [comment]
He said on discord he's going to repeat the process before merging.

**Review by blasten — APPROVED** [review]

**Pull request merged**
